src/file_format.py

```diff
@@ -42,7 +42,7 @@ def complete_path(text, state):
 readline.parse_and_bind('tab: complete')
 
 
-exec_policy = {'cpp': ' -name "*.h" -or -name "*.hpp" -or -name "*.cpp" | xargs rosrun ipa_code_refactoring clang_format -i -style=file',            # CPP
+exec_policy = {'cpp': ' -name "*.h" -or -name "*.hpp" -or -name "*.cpp" | xargs clang-format-3.8 -i -style=file',            # CPP
```

Comment:
clang-format is rather uncool with this. They do not have a binary without the version suffix. We can ask for the path or the name to  the clang executable in the script and supply a default value for it.